<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exporter</class>
 <widget class="QWidget" name="Exporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>A3Plus Exporter</string>
  </property>
  <widget class="QGroupBox" name="fileGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>79</height>
    </rect>
   </property>
   <property name="title">
    <string>File</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Terbuka :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>leDoc</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leDoc">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="tbDet">
      <property name="text">
       <string>Deteksi</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Export ke :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>leExpF</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leExpF"/>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="tbBrowse">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Pilih</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="corelGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Corel Draw</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="1" column="0">
     <widget class="QCheckBox" name="kurvaOto">
      <property name="text">
       <string>Kurva Otomatis</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Gunakan Versi</string>
      </property>
      <property name="buddy">
       <cstring>comboVersi</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboVersi">
      <property name="styleSheet">
       <string notr="true">text-align:&quot;right&quot;</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Auto</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="btPdfSetting">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Buka PDF Setting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="paramGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameter</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="5" column="4">
     <widget class="QPushButton" name="pbExport">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>File</string>
      </property>
      <property name="buddy">
       <cstring>leFile</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLineEdit" name="leBahan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="4">
     <widget class="QPlainTextEdit" name="txKet"/>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="leKlien"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Klien</string>
      </property>
      <property name="buddy">
       <cstring>leKlien</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Keterangan</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="buddy">
       <cstring>txKet</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="sideCheck">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>BB</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Qty</string>
        </property>
        <property name="buddy">
         <cstring>leQty</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leQty"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Bahan</string>
      </property>
      <property name="buddy">
       <cstring>leBahan</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Halaman</string>
      </property>
      <property name="buddy">
       <cstring>lePage</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QProgressBar" name="pBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="leFile"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lePage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="histGroup">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>10</y>
     <width>268</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Riwayat</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Filter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>leFilter</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="leFilter"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pbFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cbxViewRange">
        <item>
         <property name="text">
          <string>Hari ini</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Semua</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Lihat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>cbxViewRange</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="histTable">
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btFirst">
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btPrev">
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spHalaman">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>/1</string>
        </property>
        <property name="prefix">
         <string>Page </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btNext">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btLast">
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="pbDetach">
        <property name="text">
         <string>Layar Penuh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="status" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>405</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>432</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Tentang ?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leKlien</tabstop>
  <tabstop>leFile</tabstop>
  <tabstop>leBahan</tabstop>
  <tabstop>lePage</tabstop>
  <tabstop>leQty</tabstop>
  <tabstop>txKet</tabstop>
  <tabstop>pbExport</tabstop>
  <tabstop>leDoc</tabstop>
  <tabstop>tbDet</tabstop>
  <tabstop>leExpF</tabstop>
  <tabstop>tbBrowse</tabstop>
  <tabstop>cbxViewRange</tabstop>
  <tabstop>leFilter</tabstop>
  <tabstop>pbFilter</tabstop>
  <tabstop>histTable</tabstop>
  <tabstop>pbDetach</tabstop>
  <tabstop>comboVersi</tabstop>
  <tabstop>kurvaOto</tabstop>
  <tabstop>btPdfSetting</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>leFilter</sender>
   <signal>returnPressed()</signal>
   <receiver>pbFilter</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
